src/main/java/com/minecolonies/coremod/commands/ActionArgumentType.java

```diff
@@ -95,7 +101,7 @@ public int allowedSpaceCount()
         final List<String> colonyIdList = new ArrayList<>(colonyList.size());
         for (final IColony colony : colonyList)
         {
-            colonyIdList.add(String.valueOf(colony.getID()));
+            colonyIdList.add(String.valueOf(colony.getDimension()) + "|" + String.valueOf(colony.getID()));
```

Comment:
How does this work ingame now? Can you show us a screenshot?

Reply:
the colony id is just dimid|colonyid now (as can be seen in the colony list)

Reply:
kk